app/controllers/application_controller.rb

```diff
@@ -24,4 +24,18 @@ def authenticate_user!
   def warden
     request.env['warden']
   end
+
+  def set_search(filter: ApplicationSearchFilter.new, default_sort_by: 'submitted_at')
+    sorting = Sorting.new(
+      permitted_params[:sorting] || { sort_by: default_sort_by }
+    )
+
+    pagination = Pagination.new(current_page: permitted_params[:page] || 1)
```

Comment:
Perhaps we cold make the 1 here a constant in Pagination - I know its pretty obvious but some thing like:

```
Pagination::DEFAULT_PAGE
```

Would be clearer and not me we have a magic number here.

Reply:
👍